src/perfglue/heap_profiler.cc

```diff
@@ -79,6 +79,12 @@ bool ceph_heap_profiler_running()
 
 static void get_profile_name(char *profile_name, int profile_name_len)
 {
+#if __GNUC__ && __GNUC__ >= 8
+#pragma GCC diagnostic push
+  // Don't care, it doesn't matter, and we can't do anything about it.
```

Comment:
What I mean is, we can check the return value, but there's nothing we can do with the information since the function returns void.

(I guess we could log something, but it doesn't really seem worth it.)

Reply:
gotcha, agreed.